src/python/pants/fs/fs_test.py

```diff
@@ -7,51 +7,53 @@
 from pants.engine.console import Console
```

Comment:
The changes in this file are also prework for Stu's graph change. With that change, we can no longer inject arbitrary Params to goal rules. So, instead, here, we replace the manual injection of `MessageToGoalRule` with a singleton rule.